contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
Fedora will update 4.9.14 straight to the 4.10.3 which is already in `updates-testing` repo, so most probably this will not become an issue at all because as I said before 1.8 will be in f26 which will be shipped with 4.10 from the beginning and f25 will stick to the 1.7.